src/lib/cli_lib/cli_lib.ml

```diff
@@ -138,6 +138,7 @@ let write_keypair_exn {Keypair.private_key; public_key; _}
       Writer.close pubkey_f
   | Error e -> raise (Error.to_exn e)
 
+(** Reads a private key from [privkey_path] using [Secret_file] *)
```

Comment:
did you move that or did ocamlformat?

Reply:
I moved it -- it wasn't hugging the function so ocamlformat didn't know this doccomment was meant for that function

Reply:
oops